main/glossary/README.md

```diff
@@ -8,24 +8,34 @@ not be trusted as an accurate depiction, since it is provided by
 the maker of the mint and could be deceptive, but is useful for debugging and double-checking.
 
 The AllegedName must be a string.
-
+<a name="AmountMath"></a>
 ## AmountMath
-AmountMath executes the logic of how amounts are changed when digital assets are merged, separated, or otherwise manipulated. For example, a deposit of 2 bucks into a purse that already has 3 bucks gives a new balance of 5 bucks. An empty purse has 0 bucks. AmountMath relies heavily on polymorphic MathHelpers, which manipulate the unbranded portion.
+AmountMath executes the logic of how amounts are changed when digital assets are merged, separated,
+or otherwise manipulated. For example, a deposit of 2 bucks into a purse that already has 3 bucks 
+gives a new balance of 5 bucks. An empty purse has 0 bucks. AmountMath relies heavily on polymorphic
+[MathHelpers](#MathHelpers), which manipulate an `amount`'s [extent](#Extents) or unbranded portion.
+See the [ERTP Guide's AmountMath section](https://agoric.com/documentation/ertp/guide/amount-math.html) for more information.
 
+<a name="Amounts"></a>
 ## Amounts
 Amounts are the canonical description of tradable goods. They are manipulated
 by issuers and mints, and represent the goods and currency carried by purses and
 payments. They can be used to represent things like currency, stock, and the
 abstract right to participate in a particular exchange.
 
-An amount is composed of a `Brand` with an `Extent`.
+An amount is comprised of a [brand](#Brands) with an [extent](#Extents).
+
+See the [ERTP Guide's Amounts section](https://agoric.com/documentation/ertp/guide/amounts.html) for more information.
 
 ## AssetHolder
 Purses and Payments are AssetHolders.
-
+<a name="Brands"></a>
```

Comment:
I can see duplicating the anchor declaration in cases where you want singular and plural links to both work.